contracts/tests/test_auction.py

```diff
@@ -18,18 +20,30 @@
     setupAuctionApp,
 )
 
+# min balance: 0.1 ALGO
+# smart contract, base: 0.1 ALGO
+# smart contract, bytes: 0.05 * 2 = 0.1 ALGO
+# smart contract, ints: 0.0285 * 8 = 0.228 ALGO
+# smart contract, escrow: 0.1 + 0.1 + 0.003 = 0.203 ALGO (min, nft, txns)
+# smart contract, create + setup txns: 0.001 * 2 = 0.002 ALGO
+CREATOR_FUNDS = 100_000 + 100_000 + 100_000 + 228_000 + 203_000 + 2000
```

Comment:
See my comment about `fundAccount`. Also an approximate for how much is needed by the creator account.